homeassistant/components/xiaomi_miio/fan.py

```diff
@@ -139,6 +131,7 @@
 OPERATION_MODES_AIRPURIFIER_2S = ["Auto", "Silent", "Favorite"]
 PRESET_MODES_AIRPURIFIER_2S = ["Auto", "Silent", "Favorite"]
 OPERATION_MODES_AIRPURIFIER_3 = ["Auto", "Silent", "Favorite", "Fan"]
+PRESET_MODES_AIRPURIFIER_3C = ["Auto", "Silent", "Favorite"]
```

Comment:
Do we still need the operation mode constants here?

Reply:
Of course not, I forgot about these consts.

Reply:
Removed

Reply:
Can we already use EntityDescription the fan platform to support translations?

Reply:
This question doesn't block this PR, right?

Reply:
Right, there is no way to translate fan modes names via EntityDescription.